server/store/apistore/poll_test.go

```diff
@@ -0,0 +1,79 @@
+package apistore
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin/plugintest"
+	"github.com/matterpoll/matterpoll/server/utils/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPollStoreGet(t *testing.T) {
+	t.Run("all fine", func(t *testing.T) {
+		api := &plugintest.API{}
+		api.On("KVGet", testutils.GetPollID()).Return(testutils.GetPoll().EncodeToByte(), nil)
+		store := setupTestStore(api)
```

Comment:
Should call `defer api.AssertExpectations(t)` in all test cases.

Reply:
Good catch! :+1: